cli.js

```diff
@@ -13,186 +13,192 @@ var localCLI = resolveCwd('ava/cli');
 // see https://github.com/nodejs/node/issues/6624
 if (localCLI && path.relative(localCLI, __filename) !== '') {
 	debug('Using local install of AVA');
-	require(localCLI);
-	return;
+	require(localCLI); // eslint-disable-line import/no-dynamic-require
+} else {
+	load();
 }
```

Comment:
Instead of nesting the below in a function, maybe we should split it out into a new file and just have this file as a conditional loader CLI?

```js
if (localCLI && path.relative(localCLI, __filename) !== '') {
	debug('Using local install of AVA');
	require(localCLI); // eslint-disable-line import/no-dynamic-require
} else {
	require('./cli');
}
```

And rename this file to `_cli.js` or something.